### DISCUSSION TITLE
ch03_06 질문입니다.

### DISCUSSION THREAD

**Discussion by wand96** [tedilabs/fastcampus-devops#65] [Q&A] [answered]
![캡처](https://user-images.githubusercontent.com/84069743/167968452-59af88ff-2e51-4425-96b5-0ddf8cda11df.PNG)

**Comment by posquit0** [comment, marked as answer]
안녕하세요, 해당 pem 파일은 본인의 AWS 계정에 설정한 EC2 서버 접근용 SSH Key 입니다.
AWS 웹 콘솔의 EC2 서비스 접속, [Network & Security] - [Key Pairs] 메뉴 접근, 이곳에 등록한 SSH Key를 사용하시면 됩니다. 만약 등록된 SSH Key가 없다면 웹 콘솔 상에서 새로운 키 등록 과정에서 생성된 키를 다운로드 받을 수 있습니다.

### 참고자료

- [SSH 키 등록 및 EC2 접속 방법](https://6161990src.tistory.com/115)
